ui/apps/platform/src/services/VulnerabilityExceptionService.ts

```diff
@@ -0,0 +1,94 @@
+import { generatePath } from 'react-router-dom';
+
+import { SlimUser } from 'types/user.proto';
+import { SearchFilter, ApiSortOption } from 'types/search';
+import { getListQueryParams } from 'utils/searchUtils';
+import { Empty } from './types';
+import axios from './instance';
+
+const baseUrl = '/v2/vulnerability-exceptions';
+
+export type VulnerabilityState = 'OBSERVED' | 'DEFERRED' | 'FALSE_POSITIVE';
+
+export type ExceptionStatus = 'PENDING' | 'APPROVED' | 'DENIED' | 'APPROVED_PENDING_UPDATE';
+
+type Comment = {
+    id: string;
+    message: string;
+    user: SlimUser;
+    createdAt: string; // ISO 8601
+};
+
+type Scope = {
+    imageScope: {
+        registry: string;
+        remote: string;
+        tag: string;
+    };
+};
+
+type ExceptionExpiry =
+    | {
+          expiryType: 'TIME';
+          expiresOn: string | null;
+      }
+    | {
+          expiryType: 'ALL_CVE_FIXABLE' | 'ANY_CVE_FIXABLE';
+      };
+
+export type VulnerabilityException = {
+    id: string;
+    name: string;
+    targetState: VulnerabilityState;
+    exceptionStatus: ExceptionStatus;
+    expired: boolean;
+    requester: SlimUser;
+    createdAt: string; // ISO 8601
+    lastUpdated: string; // ISO 8601
+    comments: Comment[];
+    scope: Scope;
+    deferralReq?: {
+        expiry: ExceptionExpiry;
+    };
+    fpRequest?: Empty;
+    cves: string[];
+    deferralReqUpdate?: {
+        expiry: ExceptionExpiry;
+    };
+};
+
+export function getVulnerabilityExceptionById(id: string): Promise<VulnerabilityException> {
+    const url = generatePath(`${baseUrl}/:id`, { id });
+    return axios
+        .get<{ exception: VulnerabilityException }>(url)
+        .then((response) => response.data.exception);
+}
+
+export function listVulnerabilityExceptions(
+    searchFilter: SearchFilter,
+    sortOption: ApiSortOption,
+    page?: number,
+    pageSize?: number
+): Promise<VulnerabilityException[]> {
+    const params = getListQueryParams(searchFilter, sortOption, page, pageSize);
+
+    return axios
+        .get<{ exceptions: VulnerabilityException[] }>(`${baseUrl}?${params}`)
+        .then((response) => response.data.exceptions);
+}
+
+export type CreateDeferVulnerabilityExceptionRequest = {
+    cves: string[];
+    comment: string;
+    scope: Scope;
```

Comment:
Backend `VulnerabilityException.Scope` does bring a question if there is a potential need to export `Scope` type for components, to name as `VulnerabilityExceptionScope` to disambiguate.
https://github.com/stackrox/stackrox/blob/master/proto/api/v2/vuln_exception_service.proto#L118